docs/src/development/local/ddev.md

```diff
@@ -17,6 +18,14 @@ If you're starting from scratch, first [create a project from a PHP template]({{
 
 Make sure your computer meets the [system requirements for DDEV](https://ddev.readthedocs.io/en/stable/#system-requirements).
 
+For the integration to run smoothly, you also need the following tools:
```

Comment:
We'll have to remember to remove this when we fix that dependency. Ah the joys of maintaining docs multiple places.